app/views/shared/check_answers/_separate_representation.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @legal_aid_application.separate_representation_required %>
+  <section class="print-no-break">
+    <h2 class="govuk-heading-l"><%= t(".heading") %></h2>
+    <dl id="separate-representation-required" class="govuk-summary-list govuk-!-margin-bottom-9">
```

Comment:
Can we use the [summary list](https://govuk-components.netlify.app/components/summary-list/) component here instead? I think we're doing this in most of the other partials

Reply:
Ah yes we can, done!